osquery/tables/system/darwin/ssdeep.cpp

```diff
@@ -0,0 +1,100 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include "osquery/tables/system/darwin/ssdeep.h"
+
+#include <boost/filesystem.hpp>
+#include <fuzzy.h>
+#include <osquery/filesystem.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+namespace osquery {
+namespace tables {
+
+void expandFSPathConstraints(QueryContext& context,
+                             std::string path_column_name,
+                             std::set<std::string>& paths) {
+  context.expandConstraints(
+    "path",
+    LIKE,
+    paths,
+    ([&](const std::string& pattern, std::set<std::string>& out) {
+      std::vector<std::string> patterns;
```

Comment:
I think `patterns` is not quite a good name for results of file pattern resolving.